lib/transactions/transaction.js

```diff
@@ -22,6 +23,19 @@ function createTransaction(recipientId, amount, vendorField, secret, secondSecre
 		}
   }
 
+  /* checks if the secret/secondSecret sent is an entropy */
```

Comment:
In my opinion, it'd be better to wrap this in a well named function instead of adding the above comment